setup.py

```diff
@@ -29,8 +29,10 @@ def read(*rnames):
       license='ZPL 2.1',
       description='RestrictedPython provides a restricted execution '
                   'environment for Python, e.g. for running untrusted code.',
-      long_description=(read('src', 'RestrictedPython', 'README.rst') + '\n' +
-                        read('CHANGES.rst')),
+      long_description=(read('README.rst') + '\n' +
+                        read('docs', 'install', 'index.rst') + '\n' +
+                        read('docs', 'usage', 'basic_usage.rst') + '\n' +
```

Comment:
If `README.rst` contains the link to ReadTheDocs these two lines would not be necessary.
BTW they do not contain valid ReST PyPI can understand:

`(ERROR/3) Unknown interpreted text role "py:func".`